src/vaults/VaultRouter.sol

```diff
@@ -168,8 +189,6 @@ contract VaultRouter is Auth, Multicall, Recoverable, IVaultRouter {
     function claimDeposit(IAsyncVault vault, address receiver, address controller) external payable protected {
         _canClaim(vault, receiver, controller);
         uint256 maxMint = vault.maxMint(controller);
-
-        spoke.vaultDetails(vault); // Ensure vault is valid
```

Comment:
Can you remind me why this can/should be removed?

Reply:
See https://kflabs.slack.com/archives/C07PG2EUR9C/p1754304001505969?thread_ts=1754290230.706599&cid=C07PG2EUR9C